### PR TITLE
feat(addVault): parameterize oracleAddresses based on netName

### PR DESCRIPTION
**Pull request by 0xpatrickdev**
- use different set of oracleAddresses on mainnet and test networks
- towards #33, but still not accessible by users
- supersedes #36

### CODE REVIEW

File: src/config/inter/addVault/constants.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const format = (address: string, index: number, array: string[]) => {
+  const isLast = array.length - 1 === index;
+  return isLast ? `"${address}",` : `"${address}",\n      `;
+};
+
+export const MAINNET_ORACLE_OPERATORS = [
```
[the remark this answers is not in the document]
Comment (0xpatrickdev) [reply src/config/inter/addVault/constants.ts:1]: [](Updated in [8974b54...3ef6ae1](https://github.com/DCFoundation/cosmos-proposal-builder/compare/8974b5410bfc30aca6c6b7628ab9494b2f172c90..3ef6ae14cc144b70ce069086cd0054c5a02b237e))

File: src/config/inter/__fixtures__/add-stATOM-oracles.js [hunk 2]
```diff
@@ -28,11 +28,11 @@ const getManifestCall = harden([
       timeout: 10,
     },
     oracleAddresses: [
-      "agoric1krunjcqfrf7la48zrvdfeeqtls5r00ep68mzkr",
-      "agoric19uscwxdac6cf6z7d5e26e0jm0lgwstc47cpll8",
-      "agoric144rrhh4m09mh7aaffhm6xy223ym76gve2x7y78",
-      "agoric19d6gnr9fyp6hev4tlrg87zjrzsd5gzr5qlfq2p",
-      "agoric1n4fcxsnkxe4gj6e24naec99hzmc4pjfdccy5nj",
+      "agoric10vjkvkmpp9e356xeh6qqlhrny2htyzp8hf88fk",
```
[the remark this answers is not in the document]
Comment (0xpatrickdev) [reply src/config/inter/__fixtures__/add-stATOM-oracles.js]: Updated in [8974b54...3ef6ae1](https://github.com/DCFoundation/cosmos-proposal-builder/compare/8974b5410bfc30aca6c6b7628ab9494b2f172c90..3ef6ae14cc144b70ce069086cd0054c5a02b237e)

**Comment by 0xpatrickdev** [comment]
> I gather there's no immediate requirement for an ad-hoc list for local chain testing.
> I have a few suggestions for different factoring, at your discretion.

Expect a PR for this in the next day or so. Doing a bit of factoring around this code there so feedback on that PR would be best, if it's in addition to your 2 comments on this (great feedback btw, appreciated).

File: src/config/inter/inter.tsx [hunk 3]
```diff
@@ -139,6 +143,10 @@ const Inter = () => {
         keyword: capitalize(issuerName),
         oracleBrand: issuerName,
         proposedName: issuerName,
+        oracleAddresses:
+          netName === "main"
+            ? MAINNET_ORACLE_OPERATORS
+            : EMERYNET_ORACLE_OPERATORS,
```
Comment (dckc) [inline src/config/inter/inter.tsx:149]: ok... so we sweep all the other networks under "emerynet"

handy to have gov1 etc. in there, in that case

**Review by dckc — APPROVED** [review]

File: src/config/inter/inter.tsx (hunk 3 above)
Comment (0xpatrickdev) [reply src/config/inter/inter.tsx:149]: There's room to improve this to include gov1-gov3 a3p and serve those on localhost, ollinet, xnet

**Pull request merged**